frontend/src/reducers/session.js

```diff
@@ -21,8 +21,9 @@ export default function session(state={
         isAuthenticated: false
       });
 
-    case constants.SHOW_POPOVERMENU:
-      return merge({}, state, { hasPopOverMenuOpen: action.hasPopOverMenuOpen });
+    // TODO constants.SHOW_POPOVERMENU doesn't exist, should it?
```

Comment:
nope. pls remove. We copied some of this code from `app` when we originally split the two. leftovers.